libbeat/docs/index.asciidoc

```diff
@@ -28,6 +28,8 @@ include::./dashboards.asciidoc[]
 
 include::./newbeat.asciidoc[]
 
+include::./config-file-format.asciidoc[]
```

Comment:
Yes me too. But I still would like to see this in the Platform Reference. Reason is: most community beats seem not to be documented at all, so users at least might be interested in some common knowledge regarding all beats.